.drone.yml

```diff
@@ -6009,7 +6009,119 @@ depends_on:
   - build-tezedge-binaries
   - build-old-tezedge-binaries
   - corr/real-time-environment-cyclictest-latencies
+---
+##############################################################################################################
+# This pipeline calculates the node's p2p latency
```

Comment:
@mambisi @jurajselep 
here is "node's latency",
but step is named "perf/node-latency",
but you use binary which is called: `storage-metrics`,
so could you please clean that all up, 
that we exactly now what does this test do?
so at least, please name stuff correctly,
and please add here a short description, how does it work, like for example check: `wrk-performance-test-tezedge-new-vs-octez`

Reply:
Ohk